lib/manager/components/HomeProjectDropdown.js

```diff
@@ -39,10 +29,13 @@ export default class HomeProjectDropdown extends Component<Props, State> {
       user,
       visibleProjects
     } = this.props
-    const {profile} = user
-    const abbreviatedProjectName = getAbbreviatedProjectName(activeProject)
     const isAdmin = user.permissions && user.permissions.isApplicationAdmin()
+    const {profile} = user
     if (!profile) return null
+    const abbreviatedProjectName = getAbbreviatedProjectName(activeProject)
+    const options = visibleProjects.map(
+      project => ({value: project.id, label: project.name})
+    )
```

Comment:
I added this `options` variable so that we could do `options.find()` to locate the selected value. Arguably, one might want to just transform `activeProject` like you had previously done, but I find it cleaner to reference an already available option.